File: Web-Client/restaurant-manager/src/app/models/food.ts
import { Category } from "./category";
import { IBaseModel } from "app/models/base.model";

export class Food implements IBaseModel {
    id: string;
    name: string;
    description: string;
    price: number;
    category: Category;
    categoryId: string;
    picture: string;
}

File: src/Server/Restaurant.Server.Api/Models/Food.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace Restaurant.Server.Api.Models
{
	public class Food : BaseEntity
	{
		[Required]
		public string Name { get; set; }

		public string Description { get; set; }

		public string Recept { get; set; }

		[Required]
		public decimal Price { get; set; }

		[Required]
		public string Picture { get; set; }

		public string Currency { get; set; }

		public Guid CategoryId { get; set; }

		public virtual Category Category { get; set; }
	}
}